### ISSUE TITLE
User kill app, notification not.

### ISSUE THREAD

**Issue by expmaestro** [homerours/cordova-music-controls-plugin#162]
Hi i use this plugin in Android app now. When user or OS kill app notification does not disappear.

**Comment by tamsel12** [comment]
Yes , The same issue also i have faced , Has anyone solved this issue?

**Comment by WASDLauterbach** [comment]
I had the same problem and i've created a small workaround. 

`    window.addEventListener('beforeunload', () => {
      this.musicControls.destroy();
    });
`

Just call the EL somewhere in your logic and it will destroy the music control while killing the app.

**Comment by merbin2012** [comment]
All known issues in this plugin have been fixed and the project is now actively maintained here:
https://github.com/merbin2012/cordova-codeplay-music-controls

This repository is a fork of this plugin with fixes applied based on real production usage in my own Cordova / Capacitor projects (including Android 13/14 compatibility, seek & progress fixes, and crash resolutions).

If you find any remaining issues or have suggestions, please feel free to report them in the new repo.
Happy to collaborate and maintain this together going forward.

Thanks to the original author for the initial work.
